scraper-pipeline/extract.py

```diff
@@ -115,4 +115,4 @@ def _body_formatter(self, response: requests.Response) -> str:
 
     def _get_news_outlet(self) -> str:
         '''Returns the name of the outlet being extracted from'''
-        return "Express"
+        return " Daily Express"
```

Comment:
you have a space before Daily
```suggestion
        return "Daily Express"
```